convert2rhel/checks.py

```diff
@@ -706,106 +705,110 @@ def is_loaded_kernel_latest():
         )
         return
 
-    reposdir = get_hardcoded_repofiles_dir()
+    cmd = [
+        "repoquery",
+        "--setopt=exclude=",
+        "--quiet",
+        "--qf",
+        "C2R\\t%{BUILDTIME}\\t%{VERSION}-%{RELEASE}\\t%{REPOID}",
```

Comment:
Note for future me: This will conflict with the changes made in https://github.com/oamg/convert2rhel/pull/713